modular_skyrat/modules/goofsec/code/projectiles.dm

```diff
@@ -201,3 +201,279 @@
 	damage = 60
 */
 
+/// Bullet security pistol.
+/obj/item/gun/ballistic/automatic/pistol/security
+	name = "security pistol"
```

Comment:
Give it a unique name to fall in with our other standards.

Reply:
Will get a better name for it.